pkg/backup/schedule_exec.go

```diff
@@ -362,6 +363,11 @@ func (e *scheduledBackupExecutor) backupSucceeded(
 		e.metrics.RpoMetric.Update(details.(jobspb.BackupDetails).EndTime.GoTime().Unix())
 	}
 
+	if details.(jobspb.BackupDetails).SpecificTenantIds != nil {
+		e.metrics.RpoTenantMetric.Update(map[string]string{"tenant_id": details.(jobspb.BackupDetails).SpecificTenantIds},
```

Comment:
`SpecificTenantIDs` is a `[]string`, so you'll want to loop over them and emit a metric for each one.